### ISSUE TITLE
`import ysoisochone` fails because there's no `simps` in `scipy.integrate`

### ISSUE THREAD

**Issue by warrickball** [DingshanDeng/ysoisochrone#1]
As a cursory check ahead of editors being ready to start a JOSS review, I just tried to install `ysoisochrone` by downloading and unpacking the `.tar.gz` archive of version `0.3.1-beta` then running `pip install .`. The installation runs but I can't import `ysoisochrone` because of the following error:
```
$ python3 -c "import ysoisochrone"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/wball/work/joss-reviews/ysoisochrone/ysoisochrone-0.3.1-beta/ysoisochrone/__init__.py", line 10, in <module>
    from . import bayesian
  File "/home/wball/work/joss-reviews/ysoisochrone/ysoisochrone-0.3.1-beta/ysoisochrone/bayesian.py", line 5, in <module>
    from scipy.integrate import simps
ImportError: cannot import name 'simps' from 'scipy.integrate' (/home/wball/work/joss-reviews/ysoisochrone/lib64/python3.12/site-packages/scipy/integrate/__init__.py). Did you mean: 'simpson'?
```
I'm using Python 3.12.6 on Fedora 40 in a fresh virtual environment. I've ended up with SciPy 1.14.1 and note that `simps` was renamed to `simpson` in [SciPy 1.6.0](https://docs.scipy.org/doc/scipy/release/1.6.0-notes.html#scipy-integrate-improvements), though `simps` was quietly retained for some time and may have only recently been dropped. (The import was fine when I manually installed SciPy 1.13.0.)

**Comment by DingshanDeng** [comment]
Thanks for reporting this issue! In the newer version (v0.3.2.beta), the `simpson` is first been tried to import in case that users are using a newer version of `scipy`. Please try the newer released version of v0.3.2.beta.

**Comment by warrickball** [comment]
That worked for me, thanks!

Incidentally, because you've packaged your code as a Python package, you can install it without manually downloading the release archive by using, e.g.
```
$ pip install git+https://github.com/DingshanDeng/ysoisochrone@v0.3.2-beta
```
at the command line (`$` is the prompt), where `@v0.3.2-beta` can refer to any tag, branch or commit, or be omitted entirely to get the latest commit on the default branch.
